engine/flink/kafka-util/src/test/scala/pl/touk/nussknacker/engine/kafka/NamespaceUtilSpec.scala

```diff
@@ -0,0 +1,33 @@
+package pl.touk.nussknacker.engine.kafka
```

Comment:
I think, we don't need this UnitTest. Everything is already tested in Integration Tests.